x-pack/plugins/observability/public/pages/overview/overview.stories.tsx

```diff
@@ -78,17 +78,16 @@ const withCore = makeDecorator({
       },
     } as unknown as Partial<CoreStart>);
 
-    const config: ConfigSchema = {
+    const config = {
```

Comment:
What is this change about? Casting objects with `as` is just forcing typescript to ignore this type checking.

Reply:
Made `logs` optional in config schema.